primeqa/tableqa/metrics/answer_accuracy.py

```diff
@@ -0,0 +1,2 @@
+def compute_denotation_accuracy(dataset,predictions):
+    print(dataset.shape,predictions.predictions.shape)
```

Comment:
We should use a logger to print stuff.
I do not see a unit test for the metric.